test/cfa/sudoers_test.rb

```diff
@@ -0,0 +1,183 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require_relative "../test_helper"
+require "cfa/sudoers"
+require "tmpdir"
+
+describe CFA::Sudoers do
+  subject(:sudoers) { described_class.new(file_path: file_path) }
+
+  let(:file_path) { example_file }
+  let(:example_file) { File.join(DATA_PATH, "sudoers_example") }
+
+  let(:yast_user_spec) do
+    {
+      "user"     => "yast",
+      "host"     => "ALL",
+      "run_as"   => "root",
+      "commands" => ["/sbin/yast2"],
+      "tag"      => "NOPASSWD"
+    }
+  end
+
+  let(:cli_user_spec) do
+    {
+      "user"     => "cli",
+      "host"     => "ALL",
+      "run_as"   => "ALL",
+      "commands" => ["/sbin/yast"]
+    }
+  end
+
+  describe ".load" do
+    it "loads the file content" do
+      expect(described_class).to receive(:new)
+        .with(hash_including(file_path: file_path))
+        .and_call_original
+
+      file = described_class.load(file_path: file_path)
+
+      expect(file.loaded?).to eq(true)
```

Comment:
I expect here at least one test for real content of file, if it is really loaded.